goss.py

```diff
@@ -57,12 +61,17 @@
 
 
 # launch goss validate command on the file
-def check(module, test_file_path, output_format, goss_path):
-    cmd = ""
+def check(module, test_file_path, output_format, goss_path, vars_path):
+    cmd = "{0} -g {1}".format(goss_path, test_file_path)
+    # goss parent command flags
```

Comment:
I changed from the if-block because I felt this approach was simpler to reason about via a series of "if flag is necessary, add that to the command's string" statements.